config/environments/test.rb

```diff
@@ -57,4 +57,5 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+  config.open_weather_key = '866b5463b979e3ffb0a4f566137f898f'
```

Comment:
We need to avoid the key exposition. We can have some instance variables for them.

Reply:
Ok, I'm going to resolve that

Reply:
I don't know if you've solved it yet, but maybe you can use dotenv

Reply:
Thank you so much jvargas I will check that

Reply:
Yeah it works well, thanks jvargas